chain/types/index.go

```diff
@@ -4,7 +4,7 @@ type IndexValidation struct {
 	TipSetKey TipSetKey
 	Height    uint64
 
-	NonRevertedMessageCount uint64
-	NonRevertedEventsCount  uint64
-	Backfilled              bool
+	IndexedMessagesCount uint64
+	IndexedEventsCount   uint64
```

Comment:
Should we rename this to `IndexedNonRevertedMsgCount` and `IndexedNonRevertedEventsCount` to be precise?

Reply:
@akaladarshi That is an implementation detail that will confuse users. They're only asking for the indexed entries at a specific epoch here and so ofcourse expect only non-reverted tipsets back.